### PR TITLE
change name added, close #164

### LINKED ISSUE #164: Change username
on the settings page the user settings form should be shown:
1. username 
2. a submit button to save settings

### PR DESCRIPTION
**Pull request by ArtemKrayevskiy**

### CODE REVIEW

**Review by webdewduck — CHANGES_REQUESTED** [review]
form should be defined as form object

**Review by webdewduck — CHANGES_REQUESTED** [review]
On the original issue form should be on the settings page, not profile

File: templates/profile.html [hunk 1]
```diff
@@ -6,4 +6,24 @@
         <h1 class="display-5 mb-5">Your profile</h1>
     </div>
 </div>
+{% with messages = get_flashed_messages() %}
+
+{% if messages %}
```
Comment (webdewduck) [inline templates/profile.html]: unexpected flash messages duplicates

**Review by webdewduck — CHANGES_REQUESTED** [review]

File: index.py [hunk 2]
```diff
@@ -128,8 +128,17 @@ def profile():
     return render_template("profile.html", library=library)
 
 
-@app.route('/settings')
+@app.route('/settings', methods=["GET", "POST"])
 def settings():
+    form = NewNameForm(request.form)
+    if request.method == 'POST' and form.validate():
+        username = form.newname
+        if not re.match(r"[A-Za-z0-9_-]+", username):
```
Comment (mvchn) [inline index.py]: looks like a validation after form.validate()

**Review by webdewduck — APPROVED** [review]

**Pull request merged**